src/Entity/Adherent.php

```diff
@@ -591,6 +592,13 @@ class Adherent implements UserInterface, UserEntityInterface, GeoPointInterface,
      */
     private $commitment;
 
+    /**
+     * @var ThematicCommunity[]|Collection
+     *
+     * @ORM\ManyToMany(targetEntity="App\Entity\ThematicCommunity\ThematicCommunity")
+     */
+    private $thematicCommunities;
```

Comment:
What about using `thematicCommunitiesChiefArea` as a name to distinguish from CT memberships?

Reply:
this is not really linked to areas. maybe something like `$handledThematicCommunity` ?